modules/transport-netty4/src/main/java/org/elasticsearch/transport/netty4/Netty4InternalESLogger.java

```diff
@@ -161,7 +161,7 @@ public boolean isErrorEnabled() {
 
     @Override
     public void error(String msg) {
-        logger.equals(msg);
+        logger.error(msg);
```

Comment:
This is horrific, thank you for catching this, I need to take this one out of this PR right now and bring it all the way back to the 5.4 branch which I'm not likely to do with the remainder of these changes. Are you okay with me doing that?